spec/wwtd_spec.rb

```diff
@@ -116,6 +116,28 @@ def write_default_rakefile
         result.should include "222\n"
         result.should_not include "111\n"
       end
+
+      it "properly switches versions for every command in the array" do
+        write "Rakefile", <<-EOF
+          task(:foo){ puts "Foo in \#{RUBY_VERSION}" }
+          task(:bar){ puts "Bar in \#{RUBY_VERSION}" }
+        EOF
+
+        write ".travis.yml", <<-EOF
```

Comment:
- this will write the starting empty spaces, which might lead to some weirdness down the line
 - if you use `<<-YAML` some editors will highlight/detect the language correctly